web/repos.jspf

```diff
@@ -263,8 +293,37 @@ Copyright (c) 2015, 2016, Oracle and/or its affiliates. All rights reserved.
                                 if (branch == null) {
                                     branch = "N/A";
                                 }
+                                String currentVersion = ri.getCurrentVersion();
+                                if (currentVersion == null) {
+                                    currentVersion = "N/A";
+                                }
                                 %><td><%= Util.htmlize(type) %></td><%
                                 %><td><%= Util.htmlize(parent) %> (<%= Util.htmlize(branch) %>)</td><%
+                                %><td><%
```

Comment:
I'd like to stay with the one line like this:
![screenshot from 2016-11-10 16-51-57](https://cloud.githubusercontent.com/assets/6997160/20183573/163bcbb0-a766-11e6-96fb-3cbe7719526e.png)